int32 invalid_data
---
bool result